packages/apsara-ui/src/Listing/Listing.styles.tsx

```diff
@@ -19,3 +19,89 @@ export const FilterActions = styled.span`
     display: flex;
     align-items: center;
 `;
+
+export const ScrollableList = styled.div`
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 90%;
+    height: 70%;
+    background-color: ${({ theme }) => theme.backgroundColor || "white"};
+    overflow-y: auto;
+    box-shadow: rgb(179 179 179 / 31%) 0px 1px 5px;
+`;
+
+export const UserCard = styled.div`
```

Comment:
this is only for storybook right?

it is better to have another file to separate styling required by component and styling for storybook purposes.

or another alternative is just to put this code inside the storybook.